src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,94 @@
+.wrapper {
+    min-height: 200px;
+    overflow: auto;
+    display: grid;
+    margin-top: 66px;
+    opacity: 1;
+}
+
+.rectangle {
+    grid-row: 1;
+    grid-column: 1;
+    opacity: 0.3;
+    background-color: #2A2C2F;
+    z-index: 1;
+}
+
+.wrapper:hover .rectangle {
```

Comment:
at first I have done this with multiple bgs, but this way was a bit more understandable and easy for me. Is there a problem with leaving it like this? Extra div?

Reply:
Extra element which take browser some time to render, weaknes with working with :nth childs in this case since the rectangle element is an element, but might be a pseudo-element.

As I said, that was just For Your Information